presto-docs/src/main/sphinx/release/release-0.252.rst

```diff
@@ -0,0 +1,40 @@
+=============
+Release 0.252
+=============
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix hanging join operator when spill is enabled and the probe side finishes before the hash builder starts.
+* Add file based password authenticator plugin.
```

Comment:
Move to a new section `Security Changes` - https://github.com/prestodb/presto/wiki/Release-Notes-Guidelines#order-of-sections

Reply:
Also, add at the end -

```
See :doc:`security/password-file`
```